src/main/scripts/src/app/core/dto/query.ts

```diff
@@ -20,7 +20,7 @@
 
 export interface Query {
 
-  collectionCodes?: string[];
+  collections?: string[];
```

Comment:
Why have you changed this? It follows the structure of DTO objects defined the back-end. See [JsonQuery](https://github.com/Lumeer/engine/blob/devel/api/src/main/java/io/lumeer/api/dto/JsonQuery.java). Does the search work after this change?

Reply:
I guess it doesn't if it did before.. just noticed that in constructor it takes `collections` (https://github.com/Lumeer/engine/blob/devel/api/src/main/java/io/lumeer/api/dto/JsonQuery.java#L73) so I changed it to that.